lib/shopify_cli/github/issue_url_generator.rb

```diff
@@ -2,17 +2,22 @@ module ShopifyCLI
   module GitHub
     module IssueURLGenerator
       def self.error_url(error)
-        title = "#{error.class}: #{error.message}"
+        title = "[Bug]: #{error.class}: #{error.message}"
         labels = "type:bug"
-        content = File.read(File.join(ShopifyCLI::ROOT, ".github/ISSUE_TEMPLATE.md"))
 
         # take at most 5 lines from backtrace
         stacktrace = error.backtrace.length < 5 ? error.backtrace : error.backtrace[0..4]
-        body = stacktrace.join("\n").to_s
-        output = content.gsub(/<!--Stacktrace(.|\n)*-->/, body)
-        query = URI.encode_www_form({ title: title, body: output, labels: labels })
-        url = "#{ShopifyCLI::Constants::Links::NEW_ISSUE}?#{query}"
-        url
+        stacktrace_text = stacktrace.join("\n").to_s
+        query = URI.encode_www_form({
+          title: title,
+          labels: labels,
+          template: "bug_report.yaml",
+          stack_trace: stacktrace_text,
+          os: RbConfig::CONFIG['host_os'],
```

Comment:
This is obviously very approximate, for example we won't know what version of Mac OS or Windows they're using. I wonder if there's some way of getting more information automatically.

Reply:
I saw you [recently](https://github.com/Shopify/shopify-cli/pull/1983/files#diff-d7d4fa4928dca81949807e1451999077bbd20521fdc00a3051168944357e3c56R177) used `RbConfig::CONFIG['host_os']`, I'm not sure why the change but I was basing on that. Do you think `RUBY_PLATFORM` is the wiser choice here?

I'm also wondering whether we should fill it in at all. It's better to have the information than not to have it, but if we care about e.g. Windows 10 vs Windows 11, this won't be sufficient anyway.

Reply:
In that case, we just wanted to know the kind of OS, but here it would be better to have more details. And I think it's probably more useful for us what `RUBY_PLATFORM` provides than the exact version of the OS, but it's not a strong opinion.